specification/deviceregistry/resource-manager/readme.md

```diff
@@ -41,6 +41,15 @@ These settings apply only when `--tag=package-2025-10` is specified on the comma
 input-file:
   - Microsoft.DeviceRegistry/stable/2025-10-01/deviceregistry.json
 suppressions:
+  - code: AvoidAdditionalProperties 
+    from: 
+      - deviceregistry.json 
+    where: 
+      - $.definitions.NamespaceAssetProperties.properties.eventGroups
+      - $.definitions.NamespaceAssetUpdateProperties.properties.eventGroups
```

Comment:
@mryanlo - was seeing test errors on "Swagger Model Validation" (https://github.com/Azure/azure-rest-api-specs/actions/runs/17141648651/job/48630081649?pr=36867). Is this the appropriate change or is there an error in how eventGroups are defined in the examples?

Reply:
the examples look correct. however, i don't understand why this is getting flagged. we should try not to add real valid properties to this suppressions list if possible (if you look below, we had to suppress only the "abnormal properties" such as attributes and anything that was a customer-defined dictionary/map property). Please ask the typespec team what is the reason for the failure

Reply:
I think I fixed the errors. The "asset" examples had eventGroup properties even though we only defined them in _namespaced_ assets, so I just reverted the asset examples.